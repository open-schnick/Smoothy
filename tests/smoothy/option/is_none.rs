use crate::failing_assertion;
use smoothy::{assert_that, OptionAssertion};

#[test]
fn succeeds() {
    let option: Option<()> = None;

    assert_that(option).is_none();
}

#[test]
fn fails() {
    failing_assertion!({
        let option: Option<()> = Some(());

        assert_that(option).is_none();
    });
}
