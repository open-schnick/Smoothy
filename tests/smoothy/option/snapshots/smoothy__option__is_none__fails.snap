---
source: tests/smoothy/option/is_none.rs
expression: assertion_failed_output
---
assertion failed: `(Option is None)`
           found:  Some(())
